main.go

```diff
@@ -296,18 +289,40 @@ func collectFilesToDeploy(absDeployPth string, config Config, tmpDir string) (fi
 	return filesToDeploy, nil
 }
 
+func stepNameWithIndex(stepInfo models.TestResultStepInfo) string {
+	name := stepInfo.Title
+	if len(name) == 0 {
+		name = stepInfo.ID + "@" + stepInfo.Version
+	}
+	return fmt.Sprintf("%d. Step (%s)", stepInfo.Number, name)
+}
+
 func deployTestResults(config Config) {
+	fmt.Println()
+	log.Infof("Collecting test results...")
 	testResults, err := test.ParseTestResults(config.TestDeployDir)
 	if err != nil {
 		log.Warnf("error during parsing test results: ", err)
-	} else {
-		log.Printf("- uploading (%d) test results", len(testResults))
+		return
+	}
+	if len(testResults) == 0 {
+		log.Printf("No test results found")
+		return
+	}
 
-		if err := testResults.Upload(config.AddonAPIToken, config.AddonAPIBaseURL, config.AppSlug, config.BuildSlug); err != nil {
-			log.Warnf("Failed to upload test results: ", err)
-		} else {
-			log.Donef("Success")
+	for i, result := range testResults {
+		if i == 0 {
+			log.Printf("List of test results:")
 		}
+		log.Printf("- %s (generated by the %s)", result.Name, stepNameWithIndex(result.StepInfo))
+	}
```

Comment:
This such a great idea. It will help a lot better understanding what is happening in the customer builds. ⭐